BrowserKit/Sources/ToolbarKit/ToolbarButton.swift

```diff
@@ -132,6 +132,7 @@ class ToolbarButton: UIButton,
                 view.removeFromSuperview()
             }
         }
+        layoutIfNeeded()
```

Comment:
Discussed on slack that we eventually want to revisit removing this, we already have a `TODO` to refactor out the loading so hopefully we don't need it anymore when we do. 

However, I believe this bit was already here before adding the loading spinner, @thatswinnie maybe we can double check if removing this affects any other the toolbar items too.